careers/careers/feeds.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+from datetime import date
+
+from django.contrib.syndication.views import Feed
+from django.core.urlresolvers import reverse
+from django.utils import feedgenerator
+
+from .utils import get_all_categories, get_all_positions
+
+
+class LatestPositionsFeed(Feed):
+    feed_type = feedgenerator.Rss201rev2Feed
+    title = 'Current Mozilla job openings'
+    description = ('The current list of job openings, available internships '
+                   'and contract opportunities at Mozilla.')
+    feed_copyright = ('Portions of this content are ©1998–%s by individual '
+                      'mozilla.org contributors. Content available under a '
+                      'Creative Commons license.' % date.today().year)
+
+    def link(self):
+        return reverse('careers.listings')
+
+    def feed_url(self):
+        return reverse('careers.feed')
+
+    def categories(self):
+        return get_all_categories()
+
+    def items(self):
+        return get_all_positions()
+
+    def item_title(self, item):
+        return item.title
+
+    def item_description(self, item):
+        return item.description
```

Comment:
According to the RSS spec you need to escape html in `description` so this doesn't need safe. Firefox displays the feed correctly.